public/factories/factories.js

```diff
@@ -0,0 +1,55 @@
+(function() {
```

Comment:
We will eventually refactor to a couple factories with users, events, and usersAndEvents